.gitignore

```diff
@@ -1,6 +1,7 @@
-# This is a skeleton created by zproject.
 # You can add hand-written code here.
-### Note: General ignorance of CMake files has been added in gitignore; take care upon regens
+
+build/
+ecpp/*.cpp
```

Comment:
gitignore could be less...
with cmake build is out of source tree... so, you can remove from it all temporary/build files...

Reply:
Shall be done later.